wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -47,19 +47,26 @@ <h3>{% if postelection.is_london_assembly_additional %}{% trans "Additional memb
             {% if object.election.slug == "europarl.2019-05-23"%}
                 {% include "elections/includes/eu_results.html" with card=0 %}
             {% endif %}
-            {% if postelection.election.description %}
-                <div class="ds-details">
```

Comment:
Is this correct usage of `ds-details` if you've only got one, or should it always be a `<ul>`?
https://democracyclub.github.io/design-system/components/details/

I've made it a `<ul>` here as now we might have 2, but I reckon there are other places in WCIVF where we are doing this wrong if it should always be a `<ul>`

Reply:
No need to use a `ul` when there is only one item...I think a list of one item isn't really a list? Ideally we'd be clever about wrapping it in a list if there were more than one, but I don't think this matters a lot. 

I think it might be easier to read if you put the wrapping `li` in the parent template, and make the include just contain the `details` element. This way we know that the include is 'self contained' and doesn't need to be used inside a `ul`

Reply:
makes sense. Done in https://github.com/DemocracyClub/WhoCanIVoteFor/pull/2071/commits/15084cf18dd9686af2685bfc3124e7f2fa0a4355